### DISCUSSION TITLE
Stat panel color mode based on value

### DISCUSSION THREAD

**Discussion by nikobearrr** [grafana/grafana#38146] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I would like to be able to set the color mode (value vs background) based on the value (or thresholds) I have.

**Why is this needed**:

On dashboards where we have alerts we color the values with red color. It would be easier to spot if the background is red.

Currently it looks like this:

<img width="616" alt="Screenshot 2020-08-31 at 10 01 06" src="https://user-images.githubusercontent.com/8693064/91687502-edff0880-eb70-11ea-87d4-11ad6248e1f5.png">

I want to achieve this, but only when it's in alerting mode:

<img width="633" alt="Screenshot 2020-08-31 at 10 00 57" src="https://user-images.githubusercontent.com/8693064/91687513-f3f4e980-eb70-11ea-9e85-aaa2960a6861.png">

This can be based on the value (like threshold is), or based on the threshold itself.
I have those thresholds:

0 - green
2 - yellow
4 - red

I would like green & yellow to be value colored, but the red should have background colored

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
